src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -26,6 +26,7 @@ import ThunderstoreVersion from "../../model/ThunderstoreVersion";
 import ManagerInformation from "../../_managerinf/ManagerInformation";
 import OnlineModList from "../views/OnlineModList.vue";
 import * as PackageDb from '../../r2mm/manager/PackageDexieStore';
+import { extractZippedProfileFile } from "../../utils/ProfileUtils";
```

Comment:
This is fine now but we'll likely end up importing all the stuff from ProfileUtils, so this would be cleaner as `import * as ProfileUtils`. This can be changed in the next PR.